jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -338,9 +384,13 @@ def _on_is_active_changed(self, *args):
         for mark in self.marks.values():
             mark.visible = self.is_active
 
-    def plot_selected_points(self):
+    def _table_selection_changed(self, msg={}):
         selected_rows = self.table.selected_rows
 
+        self.table_selected._clear_table()
+        for selected_row in selected_rows:
+            self.table_selected.add_item(selected_row)
```

Comment:
this is a little hacky, but improving it probably requires building more functionality into the `Table` component.

Reply:
Longer term, as we deal with large catalogs, anything we can do to dodge loops is good.